### PR TITLE
Create a makefile variable for the modwsgi user

### PR DESCRIPTION
**Pull request by fredj**
Make it easy the change the value

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl [hunk 1]
```diff
@@ -9,7 +9,7 @@ vars:
     srid: -1
 
     # the Unix user under which the modwsgi daemon processes are executed.
-    modwsgi_user: www-data
+    modwsgi_user: MODWSGI_USER
```
Comment (pvalsecc) [inline c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl]: Your are not supposed to add something to interpreted/environment for that to work?

Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl]: I thought that was not necessary since #2336

Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl]: Added `modwsgi_user` in the interpreted section bellow

**Comment by pvalsecc** [comment]
LGTM

**Pull request closed without merging**